packages/theme/src/search/responsive.less

```
@@ -0,0 +1,14 @@
+@import '../custom.less';
+@import './vars.less';
+
+@search-prefix-cls: ~'@{css-prefix}search';
+
+@media screen and (max-width: 1280px) {
+  .@{search-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word search-prefix-cls (CssSyntaxError)

(CssSyntaxError)